slowfast/models/build.py

```diff
@@ -26,6 +27,95 @@
 """
 
 
+def get_fsdp_params(cfg) -> dict:
```

Comment:
nit: `get_fsdp_args` - params gives me the sense that this is model parameters :)

Reply:
Makes sense, changed!